sdk/communication/communication-toll-free-verification/test/internal/headers.spec.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AzureKeyCredential } from "@azure/core-auth";
+import { Context } from "mocha";
+import { PipelineRequest } from "@azure/core-rest-pipeline";
+import { SDK_VERSION } from "../../src/utils/constants";
+import { TokenCredential } from "@azure/identity";
+import { TollFreeVerificationClient } from "../../src";
+import { assert } from "chai";
+import { configurationHttpClient } from "../public/utils/mockHttpClients";
+import { createMockToken } from "../public/utils/recordedClient";
+import { isNode } from "@azure/test-utils";
+import sinon from "sinon";
+
+describe("TollFreeVerificationClient - headers", function () {
+  const endpoint = "https://contoso.spool.azure.local";
+  const accessKey = "banana";
+  let client = new TollFreeVerificationClient(endpoint, new AzureKeyCredential(accessKey), {
+    httpClient: configurationHttpClient,
+  });
+  let request: PipelineRequest;
+
+  afterEach(function () {
+    sinon.restore();
+  });
+
+  it("calls the spy", async function () {
+    const spy = sinon.spy(configurationHttpClient, "sendRequest");
+    await client.getCampaignBrief("63215741-b596-4eb4-a9c0-b2905ce22cb0", "US");
```

Comment:
Is the test dependent on a campaign brief with this ID existing in the test resource? If so, I would suggest setting it as an environment variable, so it's easier to test against different resources/environments without having to make changes to the test code itself. 

Also, double check that this is not a sensitive value.

Reply:
I don't have all the context in regard to this API, so if you are sure that this ID will work with any resource on any environment, then I'd simply recommend moving it to a `const` and use that instead to prevent hardcoding it on every call to `getCampaingBrief`.

Reply:
@danielortega-msft I think this test file only validates some specifics regarding the "headers", if you look above you can see the fake endpoint and "banana" access key. I don't think the campaignBriefID is expected to exist.

Reply:
Yes, those are just dummy data for call back tests. I have added a const to avoid using this Id repetitively.